pkg/projects/dir.props

```diff
@@ -49,6 +49,11 @@
     </Otherwise>
   </Choose>
 
+  <ItemGroup>
+    <AdditionalLibPackageExcludes Condition="'$(SymbolFileExtension)' != ''" Include="%2A%2A\%2A$(SymbolFileExtension)" />
```

Comment:
Because that would trigger globbing.  This is supposed to represent the literal pattern string.  Not files that live on disk.